SagaFlow.UI/src/lib/CommandForm.svelte

```diff
@@ -0,0 +1,137 @@
+<svelte:options tag={"sf-command-form"}/>
+<script lang="ts">
+    import type {Command, Config} from "$lib/types";
+    import sagaFlow, {defaultSagaFlowServer} from "../state/SageFlowState";
+    import {onDestroy} from "svelte";
+    import ResourceSelector from "$lib/ResourceSelector.svelte";
+
+    export let serverkey: string = defaultSagaFlowServer;
+    export let commandid: string;
+
+    let commandDefinition: Command;
+    let form: HTMLFormElement;
+    let sendingCommand = false;
+    
+    let lastErrorMessage : string | undefined;
+
+    let config: Config = {
+        commands: {},
+        resourceLists: {}
+    };
+
+    const sendCommand = () => {
+        if (sendingCommand) return;
+        
+        sendingCommand = true;
+
+        const command = Object.fromEntries(new FormData(form).entries());
+        
+        sagaFlow
+            .sendCommandAsync(commandid, command, serverkey)
+            .catch(error => showError(error));
+    }
+
+    const resetCommand = () => sendingCommand = false;
+    
+    const showError = <TError>(error: TError) => {
+        console.error("Error: ", error);
+        
+        lastErrorMessage = `${error}`;
+    }
+
+    const unsubscribe = sagaFlow
```

Comment:
Your should make sagaFlow a derived store so you can use the $sagaflow syntax and have it manage subscribe and unsubscribe.

Reply:
$ syntax is actually a sveltekit concept and not available for vanilla svelte.

Reply:
No it's not, This is what i mean https://learn.svelte.dev/tutorial/auto-subscriptions

Then you can probably mix that with derived stores https://learn.svelte.dev/tutorial/derived-stores

Reply:
fixed